lib/models/cfd.dart

```diff
@@ -0,0 +1,57 @@
+import 'amount.model.dart';
+import 'order.dart';
+
+enum CfdStatus {
+  open,
+  closed,
+  failed,
+}
+
+extension CfdStatusExtension on CfdStatus {
+  static const displays = {
+    CfdStatus.open: "Opem",
+    CfdStatus.closed: "Closed",
+    CfdStatus.failed: "Failed",
+  };
+
+  String get display => displays[this]!;
+}
+
+class Cfd {
+  String customOutputDetailId;
+
+  CfdStatus status;
+
+  int liquidationPrice;
+  int openPrice;
```

Comment:
Not sure if we should bother but with `int` you will loose the decimal precision.

Reply:
This was changed to `double`, we should have probably squashed a few commits here 😬